src/components/application_manager/src/commands/mobile/show_constant_tbt_request.cc

```diff
@@ -181,6 +181,7 @@ void ShowConstantTBTRequest::on_event(const event_engine::Event& event) {
   switch (event.id()) {
     case hmi_apis::FunctionID::Navigation_ShowConstantTBT: {
       LOG4CXX_INFO(logger_, "Received Navigation_ShowConstantTBT event");
+      const char* return_info = NULL;
```

Comment:
@OHerasym I think it would be better to use `std::string` as I proposed before and check it for empty before passing to response. For example:
```
SendResponse(result, result_code, return_info.empty() ? 0 : return_info.c_str(), &(message[strings::msg_params]));
```
It's bad idea to store raw pointer to `std::string::c_str()` beacause string could be deleted any time after some logic changes and you'll get invalid pointer